tests/flashcore/test_review_logic_duplication.py

```
@@ -0,0 +1,347 @@
+
+Issue: Core review submission logic exists in two places:
+1. ReviewSessionManager.submit_review()
+2. _submit_single_review() in _review_all_logic.py
+
+This creates maintenance hazards and violates DRY principle.
+"""
+
+import pytest
+from datetime import datetime, timezone, date
+from uuid import uuid4
+from unittest.mock import MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)